### DISCUSSION TITLE
Added folder - added collection

### DISCUSSION THREAD

**Discussion by studio-express** [Icon-Shelf/icon-shelf#86] [Ideas]
When I drop a folder with icons into the program window, I expect to see the contents of the folder imported (possibly subfolders) and the collection automatically created by folder name.

In Version 0.1.11 (0.1.11) I first create the collection, then specify the folder with the icons. And so for each collection and folder.

For example, I have 100 folders with icons. I need to specify everything manually for each collection. If a folder has a subfolder? ...

**You have a great program! I hope I gave you a good idea for future updates!**

**Comment by MrRobz** [comment]
Thank you for your kind words. I think your idea is really nice.

**Comment by snimavat** [comment]
+1 for the feature.
